### DISCUSSION TITLE
settare pulsante spia di allarme in base a condizioni

### DISCUSSION THREAD

**Discussion by mattnie** [indomus/forum#180] [HUB: Home Assistant]
Buondì a tutti, sono relativamente nuovo nel mondo Home Assistant. Sto cercando tramite un sensore di temperatura Sonoff TH16 di impostare sulla plancia principale una spia di alta temperatura.
Mi basterebbe che il "bottone" sia verde sotto una soglia e rosso sopra la soglia.
Ho fatto diverse prove ma non riesco. Potete per favore aiutarmi?
Grazie e saluti

**Comment by Larry-78** [comment]
Ciao ti consiglio di utilizzare button-card ci sono degli esempi che faranno di sicuro al caso tuo. Potrai crearne uno in base allo stato della tua entità, per esempio puoi fagli cambiare colore o addirittura farlo lampeggiare se la temperatura supera la soglia impostata. Non voglio scoraggiarti visto che sei alle prime armi, ci sarebbe da studiarselo un po' visto che non è subito intuitivo, io come te ho fatto diversi tentativi prima di ottenere il risultato sperato. Buon lavoro
[https://github.com/custom-cards/button-card](url)

**Reply by mattnie** [reply]
Grazie mille, al momento ho fatto installazione manuale, ma non ho ootenuto ancora nessun risultato. Piano piano proverò, ma temo mi sfugga qualcosa.
Grazie e saluti

**Reply by Larry-78** [reply]
Prova a ricaricare il browser o meglio svuota la cache e ricarica manualmente, dopo esserti assicurato di aver installato correttamente.

**Reply by mattnie** [reply]
Grazie mille... risolto....

ora sto battagliando con automazione..... 

Grazie
